qiling/os/os.py

```diff
@@ -299,7 +305,7 @@ def _call_api(self, name, params, result, address, return_address):
             "position": self.syscalls_counter
         })
 
-        self.ql.os.syscalls_counter += 1
```

Comment:
Any reason to move this property to `ql`?

Reply:
I did not move this property, but since we are already in the `os` object, this just removes an indirection where it would use `self.ql.os.` instead of `self.`

Reply:
Oh, got it.